zerver/lib/webhooks/git.py

```diff
@@ -170,6 +170,7 @@ def get_pull_request_event_message(
     target_branch: Optional[str] = None,
     base_branch: Optional[str] = None,
     message: Optional[str] = None,
+    *,
```

Comment:
nit: Zulip's style is that this change should happen in its own separate commit:
https://zulip.readthedocs.io/en/latest/contributing/commit-discipline.html

That way it can get its own commit message that explains the reason for it separately.

Reply:
Sure I'll rebase that change. As for applying this change to other integration, I will create open other PR for them sperately.